### ISSUE TITLE
Canada.ca theme: Resize text/contrast minimum

### ISSUE THREAD

**Issue by qanitah** [wet-boew/GCWeb#1021]
When in Firefox, the text only is zoomed at 200% (or 170%), the menu text is white text on white background., esp for smaller resolution screens (1024+ width).  It fails for colour contrast and text resize. Here is a screenshot to explain what I mean.

![snap1](https://cloud.githubusercontent.com/assets/14280062/9877008/200e20dc-5b88-11e5-9c55-9026d7a1b394.jpg)

**Comment by masterbee** [comment]
@qanitah thanks for this. We are looking into this as we speak.

**Comment by hsrudnicki** [comment]
Is this above still an issue? Either way, please update the status of this issue, whether this issue can be closed as resolved/fixed or if otherwise? Thank you

**Comment by qanitah** [comment]
This issue is still not resolved. So please keep it open.

**Comment by hsrudnicki** [comment]
Please update again the status or milestone of this, whether this remains still an issue or otherwise?  Thank you

**Comment by qanitah** [comment]
I still have this issue so it can stay open.

**Comment by zzucco** [comment]
This is happening because the menu bar with the text has a minimum width larger than the bar in the background.
A simple fix would be to add the colour to the menu bar itself.
The current css has:
`#wb-sm .menu {
    display: table;
    margin-bottom: 0;
    text-shadow: 1px 1px 1px #222;
    width: 100%; }`

Adding `background: $accent-blue;` fixes this issue.

**Comment by grifare** [comment]
The layout today is totally different and I cannot see any problems now. Are you sure this should still be kept open?
